docs/understanding-docker.md

```diff
@@ -17,274 +17,287 @@ weight = -90
 # Docker Overview
 
 Docker is an open platform for developing, shipping, and running applications.
-Docker is designed to deliver your applications faster. With Docker you can
-separate your applications from your infrastructure and treat your
-infrastructure like a managed application. Docker helps you ship code faster,
-test faster, deploy faster, and shorten the cycle between writing code and
-running code.
+Docker delivers your applications quickly by separating your applications from
```

Comment:
This paragraph is repetitive and awkward. Suggest:

"""
Docker is an open platform for developing, shipping, and running applications. Docker enables you to separate your applications from your infrastructure so you can deliver software quickly. With Docker, you can manage your infrastructure like you manage your applications. Because Docker's methodologies enable you to ship, test, and deploy code quickly, you can significantly shorten the cycle between writing code and running in in production.
"""
May need some tweaks.